cmd/relayer/validate_test.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"bytes"
+	"math/big"
+	"testing"
+
+	contracts "github.com/athanorlabs/atomic-swap/ethereum"
+	"github.com/cockroachdb/apd/v3"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+)
+
+func TestValidateRelayerFee(t *testing.T) {
+	buf := new(bytes.Buffer)
+	_, err := buf.Write([]byte(contracts.SwapFactoryABI))
+	require.NoError(t, err)
+
+	swapABI, err := abi.JSON(buf)
```

Comment:
You can just write:
```
swapABI, err := abi.JSON(strings.NewReader(contracts.SwapFactoryMetaData.ABI))
```
and then remove the `bytes.Buffer` above.  There is also `bytes.NewReader` if you ever need to turn a byte slice into a reader.